lib/mozart_fetcher/component.ex

```diff
@@ -9,14 +9,16 @@ defmodule MozartFetcher.Component do
   end
 
   defp process(config, {:ok, %HTTPoison.Response{status_code: 200, body: body}}) do
-    %Component{index: 0, id: config.id, status: 200, envelope: Envelope.build({:ok, body})}
+    ExMetrics.increment("success.component.process")
+    %Component{index: 0, id: config.id, status: 200, envelope: Envelope.build(body)}
   end
 
   defp process(_config, {:error, %HTTPoison.Error{reason: reason}}) do
+    ExMetrics.increment("error.component.process")
```

Comment:
This is a bit off topic from this PR. If we're making this compatible with requester then we should return components back with non 200 statuses which are likely to be empty envelopes. This error matcher doesn't do this and also the success one above hardcodes the 200.